templates/api.hbs

```diff
@@ -1,17 +1,25 @@
 {{>header}}
 
-import { Express } from 'express'
+import { Express, Request, Response } from 'express'
 import passport from 'passport'
+import multer from 'multer'
 import * as t from './types'
 import * as v from '../../validation'
+{{#if (hasOperationSupportingMultipart operations)}}
+import * as f from '../../impl/helpers/{{lowerCase name}}MultipartHelper'
+{{/if}}
 import { Api } from '../../models'
 
 export default function(app: Express, impl: t.{{className name}}Api) {
 	{{#each operations}}
 	app.{{lowerCase httpMethod}}(
 		{{{stringLiteral (pathTemplate fullPath)}}},
 		{{>frag/apiSecurityRequirements}}
-		function (req, res) {
+		{{#if (hasOperationSupportingMultipart .)}}
+		f.upload.fields([{{#with (getFileProperties requestBody.defaultContent.schema.properties)}}{{#each this}}{name: {{{stringLiteral @key}}}, maxCount: 1}{{#hasMore}}, {{/hasMore}}{{/each}}{{/with}}]),
```

Comment:
That `maxCount`... there can be file arrays, which I think we're probably not handling here.